src/plugins/intel_cpu/src/nodes/executors/x64/subgraph.cpp

```diff
@@ -124,6 +124,67 @@ SubgraphExecutor::SubgraphExecutor(const std::shared_ptr<CPURuntimeConfig>& snip
 #endif
 }
 
+void SubgraphExecutor::separately_repack_input(const MemoryPtr& srcMemPtr,
+                                               const MemoryPtr& dstMemPtr,
+                                               const ov::intel_cpu::RepackedInput& repacked_input,
+                                               size_t tensor_rank) {
+    auto get_offset = [](const BlockedMemoryDescPtr& desc) {
+        return static_cast<ptrdiff_t>(desc->getOffsetPadding() * desc->getPrecision().size());
+    };
+
+    const auto* src_ptr =
+        srcMemPtr->getDataAs<const uint8_t>() + get_offset(srcMemPtr->getDescWithType<BlockedMemoryDesc>());
+    auto* dst_ptr = dstMemPtr->getDataAs<uint8_t>() + get_offset(dstMemPtr->getDescWithType<BlockedMemoryDesc>());
+
+    VectorDims dom;
+    const auto& shape = dstMemPtr->getShape().getDims();
+    OPENVINO_ASSERT(shape.size() <= tensor_rank, "Unsupported shape rank of repacking data");
+    init_parallel_domain(shape, tensor_rank, 2lu, dom);
+
+    const auto& in_strides = repacked_input.in_offsets();
+    const auto& out_strides = repacked_input.out_offsets();
+    OPENVINO_ASSERT(everyone_is(tensor_rank, in_strides.size(), out_strides.size(), dom.size()),
+                    "Unsupported shape rank of repacking data");
+
+    const auto& kernel = repacked_input.kernel<BrgemmCopyBKernel>();
+    if (tensor_rank == rank6D) {
+        parallel4d_repacking(kernel.get(), dom, in_strides, out_strides, src_ptr, dst_ptr);
+    } else {
+        parallelNd_repacking(kernel.get(), dom, in_strides, out_strides, src_ptr, dst_ptr);
+    }
+}
+
+std::vector<MemoryPtr> SubgraphExecutor::prepare_weights(const std::vector<MemoryPtr>& inMemPtrs,
```

Comment:
Super minor (don't hesitate to ignore 😄): In this function, variables are named using different styles: some are in camel case, some - in snake case. It would be great to align naming for all variables

Reply:
By the way, I noticed that too. And I really don't like it 😄 
Let's fix it here at least.